src/assets/i18n/dv.json5

```diff
@@ -0,0 +1,4765 @@
+  "community.all-lists.head": "ސަބްކޮމިއުނިޓީ އަދި ކަލެކްޝަން",
+  "community.search.results.head": "ސަރޗް ނަތީޖާ",
+  "community.sub-collection-list.head": "މި ކޮމިއުނިޓީގެ ކަލެކްޝަން",
+  "community.sub-community-list.head": "މި ކޮމިއުނިޓީގެ ކޮމިއުނިޓީ",
+  "cookies.consent.accept-all": "ހުރިހާ އެއްޗަކަށް އެއްބަސްވޭ",
+  "cookies.consent.accept-selected": "ސިލެކްޓްކުރި އެއްޗަކަށް އެއްބަސްވޭ",
+  "cookies.consent.app.opt-out.description": "މި އެޕް ޑިފޯލްޓްކޮށް ލޯޑްކުރަނީ (އެކަމަކު އޮޕްޓް އައުޓްކުރެވޭނެ)",
+  "cookies.consent.app.opt-out.title": "(އޮޕްޓް-އައުޓް)",
+  "cookies.consent.app.purpose": "ބޭނުން",
+  "cookies.consent.app.required.description": "މި އެޕްލިކޭޝަން އަބަދުވެސް ބޭނުންވޭ",
+  "cookies.consent.app.required.title": "(އަބަދުވެސް ބޭނުންވޭ)",
+  "cookies.consent.update": "ފަހުން ޒިޔާރަތްކުރި ފަހުން ބަދަލު އައިސްފަ، ކޮންސެންޓް އަޕްޑޭޓްކުރޭ.",
+  "cookies.consent.close": "ބަންދުކުރޭ",
+  "cookies.consent.decline": "ޑިކްލައިންކުރޭ",
+  "cookies.consent.decline-all": "ހުރިހާ އެއްޗެއް ޑިކްލައިންކުރޭ",
+  "cookies.consent.ok": "މަސައްކަތް ރަނގަޅު",
+  "cookies.consent.save": "ސޭވްކުރޭ",
+  "cookies.consent.content-notice.description": "ތިބާގެ ޕަރސަނަލް މަޢުލޫމާތުތައް މި ބޭނުންތަކަށް އެއްކޮށް ޕްރޮސެސްކުރާނެ: {purposes}",
+  "cookies.consent.content-notice.learnMore": "ކަސްޓަމައިޒްކުރޭ",
+  "cookies.consent.content-modal.description": "ތިބާއާއި ބެހޭ މައުލޫމާތު މިތަނުން ފެންނާނެ އަދި ކަސްޓަމައިޒްކުރެވޭނެ.",
+  "cookies.consent.content-modal.privacy-policy.name": "ޕްރައިވެސީ ޕޮލިސީ",
+  "cookies.consent.content-modal.privacy-policy.text": "އިތުރު މައުލޫމާތު ހޯދަން، އަހަރުމެންގެ {privacyPolicy} ކިޔާލާ.",
+  "cookies.consent.content-modal.no-privacy-policy.text": "",
+  "cookies.consent.content-modal.title": "އަހަރުމެން އެއްކުރާ މައުލޫމާތު",
+  "cookies.consent.app.title.authentication": "އޮތެންޓިކޭޝަން",
+  "cookies.consent.app.description.authentication": "ތިޔަބޭފުޅާ ސައިންއިންވުމަށް ބޭނުންވޭ",
+  "cookies.consent.app.title.preferences": "ޕްރިފަރެންސް",
+  "cookies.consent.app.description.preferences": "ތިޔަބޭފުޅާގެ ޕްރިފަރެންސް ސޭވްކުރަން ބޭނުންވޭ",
+  "cookies.consent.app.title.acknowledgement": "އެކްނޮލެޖްމަންޓް",
+  "cookies.consent.app.description.acknowledgement": "ތިޔަބޭފުޅާގެ އެކްނޮލެޖްމަންޓް ސޭވްކުރަން ބޭނުންވޭ އަދި ކޮންސެންޓްކުރޭ",
+  "cookies.consent.app.title.google-analytics": "ގޫގުލް އެނެލިޓިކްސް",
+  "cookies.consent.app.description.google-analytics": "އަހަރުމެންނަށް ސްޓެޓިސްޓިކަލް ޑާޓާ ޓްރެކްކުރުމަށް ހުއްދަދޭ",
+  "cookies.consent.app.title.google-recaptcha": "ގޫގުލް ރީކެޕްޗާ",
+  "cookies.consent.app.description.google-recaptcha": "ރެޖިސްޓްރޭޝަންކުރުމުގެ ތެރޭގަިއަދި ޕާސްވޯރޑް ރިކަވަރީކުރަމުން އަހަރުމެން ގޫގްލް ރީކެޕްޗާ ސަރވިސް ބޭނުންކުރާނަން",
+  "cookies.consent.purpose.functional": "ފަންކްޝަނަލް",
+  "cookies.consent.purpose.statistical": "ސްޓެޓިސްޓިކަލް",
+  "cookies.consent.purpose.registration-password-recovery": "ރެޖިސްޓްރޭޝަންކުރުމާއި ޕާސްވޯރޑް ރިކަވަރީކުރުން",
+  "cookies.consent.purpose.sharing": "ހިއްސާކުރަނީ"
```

Comment:
This line is missing the trailing comma (`,`).  If you add a comma to the end of this line, that should fix the formatting errors.